### DISCUSSION TITLE
Umgang mit Flügelzügen und Kurswagen verbessern

### DISCUSSION THREAD

**Discussion by marhei** [Traewelling/traewelling#2027] [Ideas / Feature Requests]
Der aktuelle Umgang von Träwelling mit Flügelzügen und Kurswagen ist ausbaufähig.

### Flügelzüge
Aktuell werden Flügelzüge an Bahnhöfen nach der Vereinigung weiterhin als zwei Züge in der Abfahrtstafel angezeigt, obwohl diese defacto ein Zug sind. Beispiel in Darmstadt mit der RB 67 und 68, die in Neu-Edingen/Friedrichsfeld vereinigt werden. Zusätzlich besteht das Problem, dass man keine Benachrichtigung für Personen mit Check-Ins im gleichen Zug, aber jeweils anderen Zugteil erhält.

Darstellung in Träwelling
<img width="544" alt="D" src="https://user-images.githubusercontent.com/10304183/211006762-04e82850-ef67-4162-b25c-00530a30e289.png">

Darstellung auf bahn.de
<img width="745" alt="Capture d’écran 2023-01-06 à 12 47 05" src="https://user-images.githubusercontent.com/10304183/211006823-741d86d5-1ea6-4e75-829c-9fd349e1f9a0.png">

### Kurswägen
Ein anderes Problem mit gleichem technischen Hintergrund existiert bei Kurswägen. So existieren diese auf Träwelling nur bis bzw. ab dem Bf wo diese mit dem Trägerzug verbunden bzw. getrennt werden. Dadurch ist es zum Beispiel bei Nachtzügen zum Teil sehr aufwendig mit ein seine Verbindung einzuchecken, da, obwohl man im gleichen Bett liegen bleibt, mehrere Träwelling-Checkins nötig sind (im Extremfall drei Checkins für den gleichen Kurswagen).

Darstellung in Träwelling
<img width="571" alt="Capture d’écran 2023-01-06 à 12 44 47" src="https://user-images.githubusercontent.com/10304183/211007528-32b46a5a-c2a3-4987-9ba2-93533589d5eb.png">

Darstellung auf bahn.de
<img width="744" alt="Capture d’écran 2023-01-06 à 12 45 38" src="https://user-images.githubusercontent.com/10304183/211007590-3db7a8fe-8480-403e-8b10-73d1f0665016.png">

**Comment by MrKrisKrisu** [comment]
Ich gebe zu, dass ich bei diesem Punkt etwas überfragt bin, da diese Probleme etwas "außerhalb unserer Macht" stehen. Da wir in dem Fall stumpf einfach nur die Daten aus HAFAS 1:1 nutzen und diese keine Informationen über das Flügeln von Zügen (weitere Zugteile sind manchmal in den Daten abgebildet) bereitstellt, weiß ich nicht, was wir hier machen können.

Hast du eine Idee oder einen Vorschlag dazu?

**Reply by MrKrisKrisu** [reply]
Da wir keine Daten dazu haben, dass wir das in irgendeiner Form beeinflussen könnten würde ich diese Discussion (leider) mal schließen. Auch, wenn es schön wäre, wenn wir hier was machen könnten.

**Comment by pascalwittler** [comment]
@marudor scheint das auf bahn.expert gelöst zu haben und zeigt zusammengehörige Zugteile wie in den DB Lounges mit Klammern an:

![Screenshot Flügelzug auf bahn.expert](https://github.com/Traewelling/traewelling/assets/20669380/3dd7d160-d3d5-43da-b872-94f0e845a2fe)
